### PR TITLE
test for render document

### LINKED ISSUE #2259: Add tests for renderDocument()
**Is your feature request related to a problem? Please describe.**
Currently there is no test file for `server/Document/index.test.jsx` which means the critical function `renderDocument()`'s input and outputs are not tested. 

**Describe the solution you'd like**
Add test coverage for `server/Document/index.jsx`

**Describe alternatives you've considered**
The application not blowing up and failure everywhere might be valid coverage. But the input and outputs should be covered by tests

**Testing notes**
Wouldn't require any testing from a tester.

Dev insight: The unit test being added are sufficient coverage for testing

**Additional context**
Raised as an outcome for https://github.com/bbc/simorgh/pull/2136

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #2259 

**Overall change:** _increaes testing coverage_

**Code changes:**

- _add a test for document/index.js as it does not have any tests currently_
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/server/Document/index.test.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  it('should render corretly', () => {
+    renderDocument({
+      bbcOrigin: 'https://www.test.bbc.co.uk',
+      data: { test: 'data' },
+      isAmp: false,
+      routes: ['someRoute'],
+      service: 'news',
+      url: '/',
+    }).then(document => {
+      expect(document).toMatchSnapshot();
```
Comment (dr3) [inline src/server/Document/index.test.jsx]:
```suggestion
      expect(document).toEqual('<!doctype html><html lang="en-GB"></html>');
```

**Review by dr3 — DISMISSED** [review]
Looks good :) Can you add some `expect(thing).toHaveBeenCalledWith` assertions for these functions please?
```
sheet.collectStyles
renderToString
getAssetsArray
renderToStaticMarkup
DocumentComponent
getStyleTag
```

**Comment by thekp** [comment]
@oluoluoxenfree your current implementation looks good to me, are you still going to add tests for `getAssetsArray` and `getStyleTag`?

**Review by thekp — APPROVED** [review]
nice work

**Review by FK78 — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**